library/src/main/scala/scala/scalajs/js/typedarray/DataViewExt.scala

```diff
@@ -33,10 +33,12 @@ object DataViewExt {
     @inline
     def setInt64(index: Int, value: Long,
         littleEndian: Boolean = false): Unit = {
+      // scalafmt: { maxColumn = 85 }
       val high = (value >>> 32).toInt
       val low = value.toInt
       dataView.setInt32(index + (if (littleEndian) 4 else 0), high, littleEndian)
-      dataView.setInt32(index + (if (littleEndian) 0 else 4), low,  littleEndian)
+      dataView.setInt32(index + (if (littleEndian) 0 else 4), low, littleEndian)
+      // scalafmt: { maxColumn = 80 }
```

Comment:
Formatting change is probably in wrong commit. Consider not changing the max line length.